Update FAQs_Support_Services.md (#1166)
SkylineCommunications/dataminer-docs: dataminer-overview/FAQ/FAQs_Support_Services.md

  - You don't have access to the Skyline IOC technical support team.
  - Your platform is exposed to technical & security vulnerabilities.
  - New purchased connectors added to your DataMiner platform are at risk to not be (fully) supported because an outdated DataMiner software version is in use.

>   - You have the option to retrieve support on core & connectors (no upgrades). The latter are charged as [professional credit services](https://community.dataminer.services/professional-service-credits/).

Reviewer: Just to be sure, does "the latter" refer to upgrades or to connectors? If it's upgrades, we may need to rephrase this second sentence to make sure this is clear.

Reviewer: OK like this?

```suggestion
  - You have the option to get support for the DataMiner core & connectors (except for upgrades). This is charged as [professional credit services](https://community.dataminer.services/professional-service-credits/).
```
